web-wallet/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fix Seed Phrase words size #1337
 - Fix colors on red background #1336
+- Fix Transactions table design #1309
```

Comment:
This should be the PR number, not the issue one, as far as I know.

P.S. Can you add the square brackets around your change and the other changes too? Links are not created [otherwise](https://github.com/dusk-network/rusk/blob/feature-1309/web-wallet/CHANGELOG.md)

Reply:
My understanding was that the square brackets would be added in a PR of yours, this is why I wanted to keep the same as what we had now so that refactoring all together would have been easier.

Wasn't aware of the PR vs issues thingy, will update the change log.